packages/website/src/main.tsx

```diff
@@ -35,7 +35,7 @@ const App: FC<{}> = () => {
 };
 
 createRoot(document.getElementById("app") as Element).render(
-  <HashRouter>
+  <BrowserRouter>
```

Comment:
hash router means: anchor tags (used in the markdown) don't work

Reply:
@michaelglass thanks so much for changing the router, hash routing was set up at a time when we were still on github pages and it was easier not to have to deal with a base path different from `/`, this is better.

Please keep in mind that explicit references to hash-based URL's are present throughout the app (next page / prev page navigation for example), let's make sure these are all compatible with the new routing.